package.json

```diff
@@ -90,6 +90,7 @@
     "marked": "^3.0.8",
     "page": "^1.11.6",
     "plotly.js-dist": "^2.6.3",
+    "private-ip": "^3.0.0",
```

Comment:
why adding this package? it's not used in the PR

Reply:
it should be installed, it's already on the grid client so no need to re-installed